tests/acceptance/course/printcourse-test.js

```diff
@@ -0,0 +1,96 @@
+import destroyApp from '../../helpers/destroy-app';
+import {
+  module,
+  test
+} from 'qunit';
+import startApp from 'ilios/tests/helpers/start-app';
+import {c as testgroup} from 'ilios/tests/helpers/test-groups';
+import setupAuthentication from 'ilios/tests/helpers/setup-authentication';
+//import Ember from 'ember';
+
+//const { isEmpty, isPresent, run } = Ember;
+//const { later } = run;
+
+var application;
+var fixtures = {};
+
+module('Acceptance: Course - Print Course' + testgroup, {
+  
+	beforeEach: function() {
+	    application = startApp();
+	    fixtures.user = setupAuthentication(application);
+	    server.create('school');
+	    server.create('academicYear');
+	    server.create('course', {
+		  year: 2013,
+		  school: 1,
+		  published: true,
+		  title: 'Back to the Future',
+		  cohorts: [1],
+		  topics: [1],
+		  objectives: [1],
+		  learningMaterials: [1],
+		  meshDescriptors: [1]
+		});
+		server.create('topic', {
+	      title: 'Time Travel',
+	      sessions: [1],
+	      school: 1,
+	    });
+		server.create('objective', {
+	      sessions: [1],
+	      school: 1,
+	      courses: [1],
+	      title: 'Gigawatt Conversion'
+	    });
+		server.create('user', {
+			lastName: 'Brown',
+			firstName: 'Emmet',
+		    id: 1
+		  });
+		server.create('learningMaterial',{
+	      title: 'Save the Clock Tower',			
+	      originalAuthor: 'Jennifer Johnson',
+	      filename: 'Clock Tower Flyer',	
+	      owningUser: 1,
+	      status: 1,
+	      userRole: 1,
+	      copyrightPermission: true,
+	      courseLearningMaterials: [1],
+	    });
+		server.create('courseLearningMaterial',{
+	      learningMaterial: 1,
+	      course: 1,
+	      required: false,
+	      meshDescriptors: [1]
+	    });
+		server.create('meshDescriptor', {
+	      courses: [1],
+	      name: "Flux Capacitor"
+	    });
+	   
+	  },
+
+	  afterEach: function() {
+	    destroyApp(application);
+	  }
+
+
+});
+
+test('test print course', function(assert) {
```

Comment:
Should probably be called test print course learner groups since that seems to be what it looks at.